install.sh

```diff
@@ -0,0 +1,98 @@
+#/bin/bash
+# The script requires root permissions
+
+function install_trento() {
+    . /etc/os-release
+    if [[ ! $NAME =~ "SUSE" ]]; then
+        echo "Not SUSE operating system."
+        return -1
+    fi
+    case "$VERSION_ID" in
+    "15.3")
+        repo="https://download.opensuse.org/repositories/devel:/sap:/trento/15.3/devel:sap:trento.repo"
+        ;;
+    *)
+        echo "Not supported OS release."
+        return -1;;
+    esac
+    path=${repo%/*}/
+    if zypper lr --details | cut -d'|' -f9 | grep $path; then
+        echo "$path repository alreadt exists. Skipping."
+    else
+        zypper ar $repo
+    fi
+    zypper ref
+    if which trento; then
+        echo "trento is already installed. Skipping."
+    else
+        zypper in trento
+    fi
+}
+
+function install_consul() {
+    if which consul; then
+        echo "consul is already installed. Skipping."
+        return 0
+    fi
+    # Unzip cannot be extracted of the fly, so we store the file, extract and remove.
+    zipfile=$(mktemp)
+    # FIXME! The version is hardcoded. It should take the last existing.
```

Comment:
maybe for now you could put the version into a variable, so that potentially one could override it via the environment.

Reply:
I will implement it properly in the later PRs. I doubd we should let the user do something like
`$ CONSUL_VERSION=1.9.9 ./install.sh`
In my understanding the installer is like "lean back, I'll do everything for you". I would prefer to do the best in the script and ask the user if only we completely failed.

Reply:
> In my understanding the installer is like "lean back, I'll do everything for you". (with only little interaction - yes/no questions) I would prefer to do the best in the script and ask the user if only we completely failed.

Yes, but we still want to allow for overrides, because we ourselves might need them.
I'm not saying we want to force the user to always set the variable, we should of course provide a default value!